File: HrmsAngular/src/app/models/disciplinary-action/notice-enquire.model.ts
import { NgbDateStruct } from "@ng-bootstrap/ng-bootstrap";
import { EnquireCommitty } from "./enquirecommitty.model";

export class NoticeEnquire {

    id :number;
    noticeID :number;
    empCode  :string;
    dateOfEnquiry :Date;
    dateOfEnquireNgb:NgbDateStruct;
    dateOfNoticeIssue :Date;
    dateOfNoticeIssueNgb:NgbDateStruct;
    enquireCommitty:EnquireCommitty[];
    enqEmpCode:string;
    venue :string;
    note :string;
    empNote:string;
    userID :number;
    companyID :number;
    empName:string;
    designation:string;
    department:string;

}

File: HrmsWebApiCore/WebApiCore/Models/DiciplinaryAction/NoticeEnquireModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace HRMS.Models.DiciplinaryAction
{
    public class NoticeEnquireModel
    {
    public int  ID {get;set;}
    public int NoticeID {get;set;}
    public string EmpCode  {get;set;}
    public DateTime DateOfEnquiry {get;set;}
    public DateTime DateOfNoticeIssue {get;set;}
    public List<EnquireCommittyModel> EnquireCommitty { get; set; }
    public string Venue { get; set; }
    public string EnqEmpCode { get; set; }
    public string Note { get; set; }
    public string EmpNote { get; set;}
    public int UserID {get;set;}
    public int CompanyID {get;set;}
    }
}